admin/admin-service/src/test/resources/responseJson.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "mosip.credential.request.generator",
```

Comment:
Rename this file to digitalCardStatusResponseJson.json

Reply:
taken care

Reply:
Resolved